src/eclingo/parsing/transformers/theory_parser_epistemic.py

```diff
@@ -49,23 +51,24 @@ def visit_TheoryAtom(self, atom, loc="body"):
 ####################################################################################
 
 
-def _theory_term_to_literal_adapter(element: AST) -> AST:
+def _theory_term_to_literal_adapter(reification: bool, element: AST) -> AST:
     assert len(element.terms) == 1
     new_element = copy(element)
     new_element.terms[0] = theory_term_to_literal(element.terms[0])
     return new_element
 
 
 def parse_epistemic_literals_elements(rule):
-    return ApplyToEpistemicAtomsElementsTransformer(_theory_term_to_literal_adapter)(
-        rule
-    )
+    return ApplyToEpistemicAtomsElementsTransformer(
+        False, _theory_term_to_literal_adapter
```

Comment:
Boolean trap.

Please watch this video https://www.youtube.com/watch?v=CnRkXO_a5mI

change ```False``` to  ```reification=False```